pom.xml

```diff
@@ -146,8 +129,6 @@
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/pipeline-model-definition-plugin</gitHubRepo>
     <jenkins.version>2.332.1</jenkins.version>
-    <groovy.version>2.4.12</groovy.version>
```

Comment:
None of the modules here depend on `groovy` directly, so I'm not entirely sure why this was here to start with. Either way we want to pick up the version defined in the core BOM, which is what will be used at runtime.